tests/native_tests/py35_only_tests.hy

```diff
@@ -24,3 +26,23 @@
   (assert (= {1 "x" #**d1 #**d2 2 "y"} {"a" 1 "b" 2 "c" 3 "d" 4 1 "x" 2 "y"}))
   (defn fun [&optional a b c d e f] [a b c d e f])
   (assert (= (fun #**d1 :e "eee" #**d2) [1 2 3 4 "eee" None])))
+
+
+(defn run-coroutine [coro]
+  "Run a coroutine until its done in the default event loop."""
+  (-> (get-event-loop)
+      (.run-until-complete (coro))))
+
+
+(defn test-fn/a []
+  "NATIVE: testing fn/a"
```

Comment:
These sorts of comments are not necessary. I think they exist because they were somehow desirable when we were using our previous nose-based testing infrastructure.